templates/base.html

```diff
@@ -45,97 +46,118 @@
         <link rel="apple-touch-icon" sizes="114x114" href= "{% static "img/icons/perdiem_icon_114.png" %}" />
         <link rel="apple-touch-icon" sizes="144x144" href="{% static "img/icons/perdiem_icon_144.png" %}" />
         {% block extrastyle %}{% endblock %}
+        <style>
+        #home-page {
+            position: relative;
+            background-size: 50%;
+            background-repeat: no-repeat;
+            background-position: 50%;
+            background-color: black;
+        }
+        </style>
     </head>
     <body>
-        <div class="off-canvas-wrapper">
-            <div class="off-canvas-wrapper-inner" data-off-canvas-wrapper>
-                <div class="off-canvas position-right" id="offCanvas" data-off-canvas data-position="right">
-                    <!-- Close button -->
-                    <button class="close-button" aria-label="Close menu" type="button" data-close>
-                        <span aria-hidden="true">&times;</span>
-                    </button>
+        {% if user.is_authenticated %}
+        <div class="show-for-small-only title-bar">
+            <div class="top-bar nav">
+                <div class="top-bar-left">
+                    <ul class="dropdown menu nav" data-dropdown-menu>
+                        <li class="home-logo"><a class="small-home-logo" href="{% url 'home' %}"><img class="home-logo" src="{% static "img/perdiem-logo.svg" %}"></a></li>
+                        <li><a class="small-nav-icon" href="{% url 'artists' %}"><i class="fa fa-search" aria-hidden="true"></i></a></li>
+                        <li><a class="small-nav-icon" href="{% url 'music' %}"><i class="fa fa-music" aria-hidden="true"></i></a></li>
+                        <li><a class="small-nav-icon" class="nav-icon" href="{% url 'leaderboard' %}"><i class="fa fa-bar-chart" aria-hidden="true"></i></a></li>
+                    </ul>
+                </div>
+                <div class="top-bar-right">
+                    <ul class="dropdown menu" data-dropdown-menu>
+                    {% if total_earned %}
```

Comment:
No, I just had no idea how to indent the {} tags properly when I was working on this page. 

I have a pretty good understanding of how to indent them now so I just need to go back and update everything